CODE-OF-CONDUCT.md

```diff
@@ -23,7 +23,7 @@ include:
 Examples of unacceptable behavior by participants include:
 
 * The use of sexualized language or imagery and unwelcome sexual attention or
-advances
+  advances
```

Comment:
I think it would be better if the word "advances" would be in the same line as the sentence before.
Also I would prefer it if it would be done in a seperate pull request.